core/src/main/java/org/bitcoinj/base/internal/ByteUtils.java

```diff
@@ -385,6 +391,10 @@ public static long readUint32(ByteBuffer buf) throws BufferUnderflowException {
         return Integer.toUnsignedLong(buf.order(ByteOrder.LITTLE_ENDIAN).getInt());
     }
 
+    public static int readInt32(ByteBuffer buf) throws BufferUnderflowException {
```

Comment:
There is a JavaDoc missing here (-:

Reply:
I can add it if you want to move forward with this PR.

Reply:
Yes, please. I'm curious about the exact wording when it comes to signedness (-:

Reply:
Updated JavaDoc has been pushed.

Reply:
Sorry, I accidently continued this discussion in the other helper above.